src/App.tsx

```diff
@@ -1,61 +1,139 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './types/Todo';
```

Comment:
Consider using more descriptive names for types. For example, `Todo` could be renamed to `TodoItem` and `User` could be renamed to `UserDetails`. 🤔